keyboards/atreus/keymaps/replicaJunction/keymap.c

```diff
@@ -1,78 +1,80 @@
-/*
- * Keyboard: Atreus
- * Keymap:   replicaJunction
- * Version:  2.1
- */
-
 #include QMK_KEYBOARD_H
+#include "config.h"
```

Comment:
Pretty sure you don't need to explicitly include `config.h`.

Reply:
Corrected in b524db0